test/jdk/java/lang/reflect/Field/NegativeTest.java

```diff
@@ -24,8 +24,7 @@
 /**
  * @test
  * @bug 8277451
- * @run testng/othervm -Djdk.reflect.useDirectMethodHandle=true NegativeTest
- * @run testng/othervm -Djdk.reflect.useDirectMethodHandle=false NegativeTest
+ * @run testng/othervm NegativeTest
```

Comment:
Does this still need othervm if it doesn't set system properties?

Reply:
Updated.